src/Lightbox.js

```diff
@@ -181,6 +181,7 @@ class Lightbox extends Component {
 						customControls={customControls}
 						onClose={onClose}
 						showCloseButton={showCloseButton}
+						closeTitle={this.props.closeButtonTitle}
```

Comment:
The header component now accepts `closeButtonTitle` but is still passed `closeTitle` from the lightbox